File: app/src/main/java/com/skyd/imomoe/util/dlna/dms/MediaItem.java
package com.skyd.imomoe.util.dlna.dms;

import org.fourthline.cling.support.model.DIDLObject;

public interface MediaItem {
    String ROOT_ID = "0";
    String AUDIO_ID = "10";
    String VIDEO_ID = "20";
    String IMAGE_ID = "30";

    DIDLObject.Class AUDIO_CLASS = new DIDLObject.Class("object.container.audio");
    DIDLObject.Class IMAGE_CLASS = new DIDLObject.Class("object.item.imageItem");
    DIDLObject.Class VIDEO_CLASS = new DIDLObject.Class("object.container.video");
}

File: app/src/main/java/com/skyd/imomoe/util/dlna/dms/MediaItem.kt
package com.skyd.imomoe.util.dlna.dms

import org.fourthline.cling.support.model.DIDLObject

interface MediaItem {
    companion object {
        const val ROOT_ID = "0"
        const val AUDIO_ID = "10"
        const val VIDEO_ID = "20"
        const val IMAGE_ID = "30"
        val AUDIO_CLASS = DIDLObject.Class("object.container.audio")
        val IMAGE_CLASS = DIDLObject.Class("object.item.imageItem")
        val VIDEO_CLASS = DIDLObject.Class("object.container.video")
    }
}